src/System.Management.Automation/engine/ShellVariable.cs

```diff
@@ -668,7 +668,7 @@ private void SetValue(object value)
         {
             // Check to see if the variable is writable
 
-            if ((_options & (ScopedItemOptions.ReadOnly | ScopedItemOptions.Constant)) != ScopedItemOptions.None)
+            if (IsReadOnly || IsConstant)
```

Comment:
Yes, I see -- bytewise operation vs. possibly 2 method calls. I will revert this change.